src/virtualship/expedition/space_time_region.py

```diff
@@ -38,20 +38,19 @@ def _check_lon_lat_domain(self) -> Self:
             raise ValueError("minimum_depth must be less than maximum_depth")
         return self
 
-
 class TimeRange(BaseModel):
     """Defines the temporal boundaries for a space-time region."""
 
-    start_time: datetime
-    end_time: datetime
+    start_time: datetime | None = None
+    end_time: datetime | None = None
 
     @model_validator(mode="after")
     def _check_time_range(self) -> Self:
-        if not self.start_time < self.end_time:
-            raise ValueError("start_time must be before end_time")
+        if self.start_time and self.end_time:
+            if not self.start_time < self.end_time:
+                raise ValueError("start_time must be before end_time")
         return self
```

Comment:
@VeckoTheGecko @ammedd 
While using pydantic, I had to change this to accept nonetype time, but this means that when we do `schedule.from_yaml()` for `fetch` it will not give any error, right? What should we do?

Reply:
I see. I thought that pydantic would have a way of disabling validation during the initialisation of the object, but looking further at the documentation its looking that that isn't possible ... . Longterm it would be good to have `start_time` and `end_time` not none, but perhaps thats something for a future PR